kmeans-sklearn/kmeans_image_quant.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import urllib2, cStringIO
+import io
+
+from PIL import Image
+from sklearn.cluster import KMeans
+from sklearn.metrics import pairwise_distances_argmin
+from sklearn.datasets import load_sample_image
+from sklearn.utils import shuffle
+
+# This function utilizes a modified version of the scikit-learn example available here
```

Comment:
not really utilizes, right? from reading the link it is either based on or inspired by?

Reply:
inspired is a better choice, will fix.